### DISCUSSION TITLE
New gui configuration

### DISCUSSION THREAD

**Discussion by michaelthompson1991** [mampfes/hacs_waste_collection_schedule#2270] [Q&A]
Hi, I currently have this setup in yaml and it gives me a calendar with everything on. Im trying to setup in the ui since the update.

I have 2 bin collections and I’m struggling what to put on which screen. I understand it says I can leave the first page and use the default settings but it’s the last page which is confusing me.

Im not sure whether i should leave the first page or not and Im not sure what to do with the last page. I tried just leaving the last page blank but the only thing it gave me was a sensor which said on or off.

Can you help please?

**Comment by 5ila5** [comment]
Answered here: https://github.com/mampfes/hacs_waste_collection_schedule/issues/2263#issuecomment-2229257405
